plugin/executor/containerd/containerd.go

```
@@ -52,10 +66,23 @@ func (e *Executor) Create(id string, spec specs.Spec, stdout, stderr io.WriteClo
 	ctx := context.Background()
 	err := e.client.Create(ctx, id, &spec, &opts)
 	if err != nil {
-		return err
+		status, _ := e.client.Status(ctx, id)
```

Why ignore error from Status?